### ISSUE TITLE
Hardening immudb installation and files

### ISSUE THREAD

**Issue by vchain-us-mgmt** [codenotary/immudb#126]
Create guidelines on how to harden the installation of immudb and protect its files, for different OSes

**Comment by zaza81** [comment]
@byo do you think this is still valid?

**Comment by byo** [comment]
@zaza81 Yes, I think we should still improve our documentation regarding that.

Currently the quick-start and installation guide do not require validation of downloaded binaries, neither with sha256 checksum validation nor with vcn authentication.

We could also create some guidelines on how to setup a kubernetes deployment with read-only-root-fs and minimal permissions.
